packages/core-snapshots/src/contracts/codec.ts

```diff
@@ -0,0 +1,10 @@
+export interface Codec {
+    blocksEncode(block: any): Buffer;
```

Comment:
Flip the naming. We are `encoding blocks` which translates to `encodeBlocks`.

Reply:
I used this naming to get endcode function like this: encoder[table_name]Encode. 
Table name is in lower case.